### PR TITLE
chore(showcase): Adding Mission42 to sites.yml

### PR DESCRIPTION
**Pull request by simonfranzen**
## Description

Added Mission42 to sites.yml

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -5158,3 +5158,18 @@
   built_by: Luke Whitehouse
   built_by_url: https://twitter.com/_lukewh
   featured: false
+ - title: Mission42
+   main_url: https://mission42.zauberware.com
+   url: https://mission42.zauberware.com
+   description: >
+     Impress yourself, impress the world!
```
Comment (DSchau) [inline docs/sites.yml]: Could we keep this as a description of what the site/app actually is? i.e. remove the marketing info--keep it focused if possible!

Comment (simonfranzen) [reply docs/sites.yml]: Yes sure!

**Comment by simonfranzen** [comment]
@lannonbr I have updated the description

**Comment by pieh** [comment]
```
docs/sites.yml: SyntaxError: Document is not valid YAML (bad indentation?) (5161:2)
   5159 |   built_by_url: https://twitter.com/_lukewh
   5160 |   featured: false
 > 5161 |  - title: Mission42
        |  ^^^^^^^^^
   5162 |    main_url: https://mission42.zauberware.com
   5163 |    url: https://mission42.zauberware.com
   5164 |    description: >
```

**Pull request merged**